lib/hammer_cli_csv/content_hosts.rb

```diff
@@ -9,14 +9,14 @@
 # have received a copy of GPLv2 along with this software; if not, see
 # http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
 
-require 'hammer_cli'
-require 'json'
-require 'csv'
-require 'uri'
+require 'hammer_cli_foreman'
+require 'hammer_cli_foreman_tasks'
```

Comment:
i think it would be better for your requires to go in `lib/hammer_cli_csv.rb`. That way all commands can then have access to them as opposed to requiring them in every command file.